fixtures/17.service.elbV2-with-provisioned-concurrency..json

```diff
@@ -0,0 +1,57 @@
+{
+  "service": "canary-deployments-test",
+  "custom": {
+    "deploymentSettings": {
+      "stages": [
+        "dev"
+      ]
+    }
+  },
+  "functions": {
+    "hello": {
+      "handler": "handler.hello",
+      "events": [
+        {
+          "alb": {
+            "listenerArn": "arn:aws:elasticloadbalancing:region:account-id:listener/app/my-load-balancer/50dc6c495c0c9188/f2f7dc8efc522ab2",
+            "priority": 1,
+            "conditions": {
+              "path": "/hello",
+              "host": "example.com",
+              "method": "GET"
+            }
+          }
+        }
+      ],
+      "provisionedConcurrency": 1,
+      "deploymentSettings": {
+        "type": "Linear10PercentEvery1Minute",
+        "alias": "custom"
+      },
+      "targetAlias": {
```

Comment:
As discussed, let's document the logic here because of the way in which sls handles provisioned concurrency.

Reply:
Added to the "Further comments" section in the PR description.